components/shared/profile/edit/EditProfile.tsx

```diff
@@ -0,0 +1,451 @@
+          className="bg-black-700 text-white-100"
+          placeholder="portfolio"
+        />
+      </div>
+      {goalFields.map((field, index: number) => (
+        <>
+          <div key={field.id} className="profile-goals-wrapper">
+            <Input
+              type="checkbox"
+              color="green-500"
+              className="order-1 size-9 bg-black-700 text-white-100"
+              id={`learningGoals[${index}]CB`}
+              {...register(`learningGoals[${index}].completed` as any)}
+            />
+            <Input
+              className="order-2"
+              id={`learningGoals[${index}]`}
+              placeholder="Enter your goal"
+              {...register(`learningGoals[${index}].name` as any)}
+            />
+            <Button
+              type="button"
+              className="text-white-100"
+              onClick={() => removeGoal(index)}
+            >
+              Remove
+            </Button>
+          </div>
+        </>
+      ))}
+      <CustomButton
+        buttonType="profileButton"
+        type="button"
+        // @ts-ignore
+        onClick={() => appendGoal({ name: "", completed: false })}
+      >
+        Add Goal
+      </CustomButton>
+
+      {experienceFields.map((field, index) => {
+        return (
+          <>
+            <div className="profile-goals-wrapper" key={field.id}>
+              <Button
+                className="text-white-100"
+                type="button"
+                onClick={() => removeExperience(index)}
+              >
+                {" "}
+                Remove
+              </Button>
+              <Input
+                id={`experiences[${index}].name`}
+                type="text"
+                {...register(`experiences[${index}].name` as any)}
+              />
+            </div>
+          </>
+        );
+      })}
+      <CustomButton
+        buttonType="profileButton"
+        type="button"
+        onClick={() => appendExperience({ name: "" })}
+      >
+        Add Experience
+      </CustomButton>
+      {/* Technologies Section with Search Box */}
+      <div className="flex flex-col justify-stretch gap-2 px-3.5 py-3  ">
+        <label
+          htmlFor="technologies"
+          className="justify-start space-y-2 text-white-300"
+        >
+          Tech Stacks
+        </label>
+        <div className="flex w-full  bg-black-700 ">
+          {technologies && // eslint-disable-next-line array-callback-return
+            technologies?.map((tech: any, index) => {
+              const icon = techStackBadges.find((badge) => badge.name === tech);
+              if (icon)
+                return (
+                  <>
+                    <span
+                      key={index}
+                      className="tech-badges paragraph-3-medium shadow-custom w-1/2   flex-row justify-between bg-black-600 text-white-100"
+                    >
+                      {icon.icon({ size: 16 })}
```

Comment:
```suggestion
      <div className="flex flex-col justify-stretch gap-2 px-3.5 py-3  ">
        <label
          htmlFor="technologies"
          className="justify-start space-y-2 text-white-300"
        >
          Tech Stacks
        </label>
        <div className="flex w-full bg-black-700 py-2 ">
          <div className="flex w-1/2 flex-wrap">
            {technologies && // eslint-disable-next-line array-callback-return
              technologies?.map((tech: any, index) => {
                const icon = techStackBadges.find(
                  (badge) => badge.name === tech
                );

                if (icon)
                  return (
                    <>
                      <span
                        key={index}
                        className="tech-badges paragraph-3-medium shadow-custom flex w-20 flex-row justify-between bg-black-600 text-white-100"
                      >
                        {icon?.icon}
```

I think this will get you closer. I would really like a screenshot of what you currently had and what it was looking like.

![image](https://github.com/agordon123/git_note/assets/4108484/c7bd506f-25d3-4668-b69c-a4dd8db48d1d)